### PR TITLE
Fix standalone java test and update pom description

### PR DESCRIPTION
**Pull request by objectiser**

### CODE REVIEW

**Comment by objectiser** [comment]
@pavolloffay Could you do a quick review please?

File: tests/container/standalone/pom.xml [hunk 1]
```diff
@@ -157,7 +157,7 @@
               <target>
                 <property name="runtime_classpath" refid="maven.runtime.classpath" />
                 <echo message="runtime classpath: ${runtime_classpath}" />
-                <java classname="org.hawkular.apm.tests.server.TestTraceServer" fork="true" spawn="true">
+                <java classname="org.hawkular.apm.tests.server.ApmMockServer" fork="true" spawn="true">
```
Comment (pavolloffay) [inline tests/container/standalone/pom.xml]: Ahh I forgot this one :disappointed:

**Comment by pavolloffay** [comment]
LGTM :+1:

**Comment by objectiser** [comment]
No problem - thanks for quick review.

**Pull request merged**